examples/README.md

```diff
@@ -54,6 +54,7 @@
 
 - [`loader-airtable`](https://observablehq.observablehq.cloud/framework-example-loader-airtable/) - Loading data from Airtable
 - [`loader-arrow`](https://observablehq.observablehq.cloud/framework-example-loader-arrow/) - Generating Apache Arrow IPC files
+- [`loader-canvas-to-png`](https://observablehq.observablehq.cloud/framework-example-loader-canvas-to-png/) - Generating a map server-side as a PNG file with node, canvas and D3
```

Comment:
This description can be shortened and better match the other descriptions. Some alternatives:

1. Generating PNG from JavaScript
2. Generating PNG from Node.js
3. Generating PNG using node-canvas
4. Generating PNG

I think my preference is (3) because most of our other data loader examples assume Node.js + JavaScript in the description, so it feels anomalous to call this one out, and it’s nice to reference the supporting library (node-canvas) explicitly.

(Also, it’s “Node.js” not “node”. Maybe “Node” is okay in some contexts, but I wouldn’t use “node” unless you are using `node` to refer specifically to the command you run from the terminal.)